libcxx/test/support/compare_types.h

```diff
@@ -529,4 +529,30 @@ struct ForwardingTestObject {
   constexpr bool operator>=(const ForwardingTestObject&) const& { return false; }
```

Comment:
Comment not attached to this line. I think we're missing an update to the `__cpp_lib_concepts` feature-test macro, the paper seems to mention something.

Reply:
I wasn't sure if there were any other changes that that feature test macro might indicate, so I didn't want to update it and potentially promise something that wasn't implemented.

Reply:
Alright, it changed at https://github.com/cplusplus/draft/pull/5701/files#diff-f46371f476893d323cfcb1383e9a70f07ff1c1f0c217e22294d0bf937aae6794 from the value that libc++ had previously, so this seems safe.